pkg/didcomm/protocol/outofband/service_test.go

```diff
@@ -585,3 +664,39 @@ func newAck(pthid ...string) *model.Ack {
 
 	return a
 }
+
+type testDIDCommMsg struct {
+	errDecode error
+}
+
+func (t *testDIDCommMsg) ID() string {
+	panic("implement me")
+}
```

Comment:
I would recommend you to replace it with `gomock`

Reply:
noted - maybe in the future